KomaMRIPlots/src/ui/DisplayFunctions.jl

```diff
@@ -981,10 +981,10 @@ function plot_kspace(seq::Sequence; width=nothing, height=nothing, darkmode=fals
             "orbitRotation",
             "resetCameraDefault3d",
         ],
-    )
-    return plot_koma(p, l; config)
-end
-
+        )
+        return plot_koma(p, l; config)
+    end
+    
```

Comment:
Don't modify indentation

Reply:
fixed in commit [479f317](https://github.com/JuliaHealth/KomaMRI.jl/pull/528/commits/479f31791aa5cec0e9af61a9e83d1bf99f680cd2)

Reply:
I dont see it modified